teku/src/main/java/tech/pegasys/teku/cli/BeaconNodeCommand.java

```diff
@@ -134,50 +133,65 @@ private static class ConfigFileCommand {
       scope = ScopeType.INHERIT)
   private File configFile;
 
-  @Mixin(name = "Network")
-  private Eth2NetworkOptions eth2NetworkOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Network%n", headingKey = "Network")
+  private Eth2NetworkOptions eth2NetworkOptions = new Eth2NetworkOptions();
 
-  @Mixin(name = "P2P")
-  private P2POptions p2POptions;
+  @CommandLine.ArgGroup(validate = false, heading = "P2P%n", headingKey = "P2P")
+  private P2POptions p2POptions = new P2POptions();
 
-  @Mixin(name = "Interop")
-  private InteropOptions interopOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Validator%n", headingKey = "Validator")
+  private ValidatorOptions validatorOptions = new ValidatorOptions();
 
-  @Mixin(name = "Validator")
-  private ValidatorOptions validatorOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Deposit%n", headingKey = "Deposit")
+  private DepositOptions depositOptions = new DepositOptions();
 
-  @Mixin(name = "Deposit")
-  private DepositOptions depositOptions;
+  @CommandLine.ArgGroup(
+      validate = false,
+      heading = "Execution Layer%n",
+      headingKey = "Execution Layer")
+  private ExecutionLayerOptions executionLayerOptions = new ExecutionLayerOptions();
 
-  @Mixin(name = "Execution Layer")
-  private ExecutionLayerOptions executionLayerOptions;
-
-  @Mixin(name = "Logging")
+  @CommandLine.ArgGroup(validate = false, heading = "Logging%n", headingKey = "Logging")
   private final LoggingOptions loggingOptions = new LoggingOptions();
 
-  @Mixin(name = "Metrics")
-  private MetricsOptions metricsOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Metrics%n", headingKey = "Metrics")
+  private MetricsOptions metricsOptions = new MetricsOptions();
+
+  @CommandLine.ArgGroup(
+      validate = false,
+      heading = "Beacon Node Data%n",
+      headingKey = "Beacon Node Data")
+  private BeaconNodeDataOptions dataOptions = new BeaconNodeDataOptions();
 
-  @Mixin(name = "Data")
-  private BeaconNodeDataOptions dataOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Data Storage%n", headingKey = "Data Storage")
+  private DataStorageOptions dataStorageOptions = new DataStorageOptions();
```

Comment:
I think these should be in the same category as `Beacon Node Data` but `DataStorageOptions` is also used by other subcommands so we need to keep it as it's own class.  I think we can just move this arg group into `BeaconNodeDataOptions` without a `heading` or `headingKey` field.